Wiki/services/services-help.md

```diff
@@ -0,0 +1,48 @@
+# CMS help
+
+The CMS team supports CLEX members with computational issues: 
+
++ **Running Models**: we support several models (see navigation bar on the left) and assist with others, subject to limitations of scope and team resources. 
```

Comment:
Not sure if you want to keep "see navigation bar on the left". I guess it depends on the final format.

Reply:
thanks, let's remove it for the moment, good spotting!